.github/workflows/run-regression-tests-command.yml

```diff
@@ -34,282 +37,77 @@ on:
-
-          if [ -z "$REPORT_PATH" ]; then
-            echo "Error: No report path provided from locate-report step" >&2
-            echo "## ⚠️ LLM Evaluation Skipped" >> "$GITHUB_STEP_SUMMARY"
-            echo "No regression test report found. The tests may have failed to generate a report." >> "$GITHUB_STEP_SUMMARY"
-            exit 1
-          fi
-
-          echo "Evaluating report at: $REPORT_PATH"
-
-          # Run the evaluation script
-          OUT_JSON="$RUNNER_TEMP/llm_eval.json"
-          poetry run python src/live_tests/regression_tests/llm_evaluation/evaluate_report.py \
-            --report-path "$REPORT_PATH" \
-            --output-json "$OUT_JSON"
-
-          # If we got here, script exit 0 and produced a judgment
-          PASS=$(jq -r '.evaluation.pass' "$OUT_JSON")
-          if [ "$PASS" = "true" ]; then RES="pass"; else RES="fail"; fi
-          echo "ran=true" >> "$GITHUB_OUTPUT"
-          echo "result=$RES" >> "$GITHUB_OUTPUT"
-
-      - name: Append LLM outcome
-        if: always() && github.event_name == 'workflow_dispatch' && github.event.inputs.pr != ''
-        env:
-          EVAL_MODEL: llama3.2:3b
-        uses: peter-evans/create-or-update-comment@v4
-        with:
-          token: ${{ github.token }}
-          comment-id: ${{ steps.pr-comment-id.outputs.comment-id }}
-          edit-mode: append
-          body: |
-            > LLM Evaluation: ${{ steps.llm-eval.outputs.ran == 'true' && (steps.llm-eval.outputs.result == 'pass' && '✅ PASS' || steps.llm-eval.outputs.result == 'fail' && '❌ FAIL' || '⚠️ ERROR') || '⚠️ Did not run' }}${{ steps.llm-eval.outputs.ran == 'true' && format(' (model: {0})', env.EVAL_MODEL) || '' }}
+          # Build the inputs JSON, only including non-empty values
+          INPUTS_JSON=$(jq -n \
+            --arg target_image "${{ inputs.target_image }}" \
+            --arg control_image "${{ inputs.control_image }}" \
+            --arg command "${{ inputs.command || 'check' }}" \
+            --arg connection_id "${{ inputs.connection_id }}" \
+            --arg state_path "${{ inputs.state_path }}" \
+            '{
+              target_image: $target_image,
+              control_image: $control_image,
+              command: $command
+            } + (if $connection_id != "" then {connection_id: $connection_id} else {} end)
+              + (if $state_path != "" then {state_path: $state_path} else {} end)')
+
+          echo "Triggering regression test workflow with inputs:"
+          echo "$INPUTS_JSON" | jq .
+
+          curl -X POST \
+            -H "Authorization: Bearer $GH_TOKEN" \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            https://api.github.com/repos/airbytehq/airbyte-ops-mcp/actions/workflows/connector-regression-test.yml/dispatches \
+            -d "$(jq -n --argjson inputs "$INPUTS_JSON" '{ref: "main", inputs: $inputs}')"
```

Comment:
Don't use curl directly. Use github marketplace action instead to call the external workflow.

Reply:
Ditto elsewhere in this pr.